mivideo/.flake8

```diff
@@ -0,0 +1,7 @@
+[flake8]
```

Comment:
I really don't want to hold up merging this for any style-based disagreements, but I suppose we should start the discussion. Are you planning on maintaining separate style guidelines for your job?

Reply:
Well, you mentioned the potential conflict between my preference for camelCase and others' for snake_case, so I thought, "what the heck?"

Reply:
I'm okay with this (at least for now), marking as resolved.